code/modules/chemistry/tools/syringes.dm

```diff
@@ -116,6 +116,16 @@
 					boutput(user, SPAN_ALERT("The [src.name] is full."))
 					return
 
+				if (istype(target,/obj/item/reagent_containers/synthflesh_pustule))
+					var/obj/item/reagent_containers/synthflesh_pustule/P = target
+					P.reagents.trans_to(src, src.amount_per_transfer_from_this)
+					logTheThing(LOG_CHEMISTRY, user, "draws 5 units of reagents from [constructTarget(target,"combat")] [log_reagents(target)] with a syringe [log_reagents(src)] at [log_loc(user)].")
+					if (!P.poked)
+						P.poked = TRUE
+						SPAWN(3 SECONDS)
```

Comment:
Why not use the `process` loop?

Reply:
Removed the SPAWN and moved it to a process loop.